### ISSUE TITLE
Trident Disable (RipTide) 

### ISSUE THREAD

**Issue by CataCombs14** [ChanceSD/PvPManager#418]
**Is your feature request related to a problem? Please describe.**
-Either this is a feature and I'm missing it, but I wanted to know if its a thing. 

**Describe the solution you'd like**
Riptide tridents seem to be annoying. If this is a function that can be disabled already. Then please tell me how. I simply want to make sure its fair, but its not when people trident away when a fight happens. Aka I just want pearls to be like the only way out etc. 

So basically just how/adding a riptide config disable option or something :D

**Comment by ChanceSD** [comment]
It does exist but only on the premium version unfortunately
